### PR TITLE
fix: mount certs for Azure Stack

### PR DESCRIPTION
**Pull request by songjiaxun**
**What type of PR is this?**

/kind bug
**What this PR does / why we need it**:
Azure Stack needs certs to be mounted on the CSI containers.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,7 @@ REGISTRY ?= andyzhangx
 REGISTRY_NAME ?= $(shell echo $(REGISTRY) | sed "s/.azurecr.io//g")
 IMAGE_NAME ?= blob-csi
 IMAGE_VERSION ?= v0.12.0
+CLOUD ?= PublicAzure
```
Comment (andyzhangx) [inline Makefile:21]: `AzurePublicCloud`

Comment (songjiaxun) [reply Makefile]: made the change

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**